src/writer/logs.py

```
@@ -1,104 +1,244 @@
+    buffer, key = routing_map.add_buffer()
+    token = _logging_routing_key.set(key)
+
+    yield buffer
+
+    routing_map.remove_buffer(key)
+    _logging_routing_key.reset(token)
+
+
+class JSONFormatter(logging.Formatter):
+    def format(self, record: logging.LogRecord, **kwargs) -> str:
+        super(JSONFormatter, self).format(record)
```

Pylint (3.3.7)

[refactor] Consider using Python 3 style super() without arguments

(R1725)